src/dom.js

```diff
@@ -648,6 +648,24 @@ export function iterateCursor(iterable, cb) {
   }
 }
 
+/**
```

Comment:
Curious, why in dom.js?

Reply:
It's mainly useful for iterating over `NodeList`s, which are a DOM API. `NodeList`s are lazily evaluated, and this method loops over the items in the list without accessing the `length` property, since that would force evaluation of the full list.

Reply:
Fair enough.

Reply:
This sounds reasonable given we already have `iterateCursor` but following this logically we could end up recreating all `Array` APIs for `NodeList` here. Instead, let's:

1. Keep this helper in the extension where it's used
2. Use `iterateCursor` (less pretty)
3. Convert NodeList to array and then run `Array#map`